types/tarantool-driver/index.d.ts

```diff
@@ -0,0 +1,60 @@
+// Type definitions for tarantool-driver 3.0
+// Project: https://github.com/tarantool/node-tarantool-driver
+// Definitions by: Evgeni Zharkov <https://github.com/zharkov-eu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { EventEmitter } from "events";
+
+export interface TarantoolOptions {
```

Comment:
unfortunately, `export =` requires a single export, so you'll have to wrap the interface inside 
```ts
declare namespace TarantoolConnection {
  // interface goes here
}
```

Reply:
Wrap inside namespace in 3d3fee9
Export keyword is redundant, as I see from travis-ci, but how fix "has no default export" problem?

Reply:
Use `import TarantoolDriver = require("tarantool-driver");` instead of `import TarantoolDriver from "tarantool-driver";` in the tests